CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change history for stripes-erm-components
 
+## 3.1.0 IN PROGRESS
```

Comment:
I like to bump the version in package.json at this point. Allows you to depend on `^3.1.0` in whatever apps you use these features, and ensures you won't accidentally release a package that has this unreleased dependency.